python/ray/tune/variant_generator.py

```diff
@@ -105,7 +104,7 @@ def grid_search(values):
 def _format_vars(resolved_vars):
     out = []
     for path, value in sorted(resolved_vars.items()):
-        if path[0] in ["alg", "env", "resources"]:
+        if path[0] in ["run", "env", "resources"]:
```

Comment:
Slightly unrelated but can we add a warning so that this doesn't act silently? I could see people doing meta learning and "env" could be a config in that setting.

Reply:
Ah, it's already included in the experiment tag, the format is `$runner_$env_$formatVars` so nothing is lost here.